partiql-parser/src/main/kotlin/org/partiql/parser/internal/V1PartiQLParserDefault.kt

```diff
@@ -1509,6 +1509,21 @@ internal class V1PartiQLParserDefault : V1PartiQLParser {
             exprPathStepAllFields(null)
         }
 
+        override fun visitValues(ctx: GeneratedParser.ValuesContext) = translate(ctx) {
+            val rows = visitOrEmpty<ExprArray>(ctx.valueRow())
+            exprBag(rows)
+        }
+
+        override fun visitValueRow(ctx: GeneratedParser.ValueRowContext) = translate(ctx) {
+            val expressions = visitOrEmpty<Expr>(ctx.expr())
+            exprArray(expressions)
+        }
+
+        override fun visitValueList(ctx: GeneratedParser.ValueListContext) = translate(ctx) {
+            val expressions = visitOrEmpty<Expr>(ctx.expr())
+            exprArray(expressions)
+        }
```

Comment:
(self-review) these were missing from the parser PR. fixes some conformance test failures